dependencies.yml

```diff
@@ -0,0 +1,57 @@
+collectors:
+
+# Temporarily disabled
```

Comment:
Could this be enabled?

Reply:
If not, can you say why it's disabled and what needs to happen to enable it?

Reply:
I believe the tarballs cause the problem here

Reply:
See https://github.com/Hypnosphi/storybook/pull/5
Enabling it requires some changes in dependencies.io components